### ISSUE TITLE
Adding a version argument to download_models

### ISSUE THREAD

**Issue by E-R** [Teichlab/celltypist#54]
It would be great to have a version argument to `download_models`. This will allow writing and sharing code for reproducible analysis. (Currently, a newer model version will likely change the results of my analysis. While `force_update` is sufficient locally, it is not sufficient if I wish to share my code).

**Comment by ChuanXu1** [comment]
@E-R, good idea, needs to reframe quite a bit to achieve this option.. will keep you updated once this functionality is implemented. In the meantime, please let us know if you need to have an archived old model.
